subt/docker/robotika/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN curl -O# http://osgar.robotika.cz/subt/virtual/model/mdnet3.64.64.13.4.relu.
 RUN curl -O# http://osgar.robotika.cz/subt/virtual/model/ssd_inception_v2_201009/frozen_inference_graph.pb
 RUN curl -O# http://osgar.robotika.cz/subt/virtual/model/ssd_inception_v2_201009/cv_graph.pbtxt
 
-RUN sudo apt-get update && sudo apt install -y ros-melodic-teleop-twist-keyboard
+RUN sudo apt-get update && sudo apt install -y ros-melodic-teleop-twist-keyboard python-laser-geometry
```

Comment:
@zwn do we want to add `python-laser-geometry` to base docker image?

Reply:
I think we do. I'd also like to remove this keyboard stuff. Is anyone still using it?

Reply:
I would remove it and if anybody will need it (locally) he can install it (locally)